package-system/AWSNativeSDK/FindAWSNativeSDK.cmake.iOS

```diff
@@ -0,0 +1,312 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+include(CMakeParseArguments)
+
+set(AWSNATIVESDK_PACKAGE_NAME AWSNativeSDK)
+
+set(AWS_BASE_PATH ${CMAKE_CURRENT_LIST_DIR}/${AWSNATIVESDK_PACKAGE_NAME})
+
+# Include Path
+set(AWSNATIVESDK_INCLUDE_PATH ${AWS_BASE_PATH}/include)
+
+# Only static libs are supported for this revision
```

Comment:
Curious: Is there a plan to add dynamic libraries later or is it not supported on iOS at all?

Reply:
no unless there is strong requirement, as that is how we setup previously (not sure whether there is any limitation here)